Makefile

```diff
@@ -1,9 +1,9 @@
 SHELL := /bin/bash
 
-OPENSHIFT_VERSION ?= 4.9.12
+OPENSHIFT_VERSION ?= 4.9.15
 PODMAN_VERSION ?= 3.4.4
 BUNDLE_EXTENSION = crcbundle
-CRC_VERSION = 1.99.1
+CRC_VERSION = 1.99.2
```

Comment:
Not sure this has to be part of a bundle version updated?

Reply:
@cfergeau no, I will remove that.